### ISSUE TITLE
Hang IDE with Scanning files to index

### ISSUE THREAD

**Issue by Delagen** [zielu/GitToolBox#49]
Added about 20 GIT projects.
Opening IDE &  it hangs totally
[log.zip](https://github.com/zielu/GitToolBox/files/1359260/log.zip)

**Comment by Piskvor** [comment]
Only manifests in PhpStorm 2017.3 EAP Build #PS-173.2941.8, built on October 4, 2017 for me - previous EAP builds (and stable versions) worked fine.
[173.2941.8.zip](https://github.com/zielu/GitToolBox/files/1361926/173.2941.8.zip)

**Comment by novotnyr** [comment]
I can confirm this on IntelliJ IDEA 2017.3 EAP (173.2941.3). 

The IDE completely hangs on startup. I am attaching a thread dump.

[threadDump-20171009-160457.txt](https://github.com/zielu/GitToolBox/files/1368737/threadDump-20171009-160457.txt)

**Comment by zielu** [comment]
Fixed in 173.0.1

**Comment by Piskvor** [comment]
Sorry, still hangs.
```
PhpStorm 2017.3 EAP
Build #PS-173.2941.8, built on October 4, 2017
PhpStorm EAP User
Expiration date: November 3, 2017
JRE: 1.8.0_152-release-1012-b1 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.10.0-37-generic
```

**Comment by zielu** [comment]
@Piskvor would you mind testing new build ?
https://www.dropbox.com/s/v6nsmqtr7f5nnql/GitToolBox-173.0.2-snapshot-49.zip?dl=0

**Comment by zielu** [comment]
Fixed in 173.0.2
